sympy/assumptions/ask.py

```diff
@@ -9,66 +9,1114 @@
+    def infinitesimal(self):
+        """
+        See documentation of ``Q.zero``.
+        """
+        return Predicate('infinitesimal')
+
+    @memoize_property(predicate_storage)
+    def positive(self):
+        r"""
+        Positive real number predicate.
+
+        ``Q.positive(x)`` is true iff ``x`` is real and `x > 0`, that is if ``x``
+        is in the interval `(0, \infty)`.  In particular, infinity is not
+        positive.
+
+        A few important facts about positive numbers:
+
+        - Note that ``Q.nonpositive`` and ``~Q.positive`` are *not* the same
+          thing. ``~Q.positive(x)`` simply means that ``x`` is not positive,
+          whereas ``Q.nonpositive(x)`` means that ``x`` is real and not
+          positive, i.e., ``Q.positive(x)`` is logically equivalent to
+          `Q.negative(x) | Q.zero(x)``.  So for example, ``~Q.positive(I)`` is
+          true, whereas ``Q.nonpositive(I)`` is false.
+
+        - See the documentation of ``Q.real`` for more information about
+          related facts.
+
+        Examples
+        ========
+
+        >>> from sympy import Q, ask, symbols, I
+        >>> x = symbols('x')
+        >>> ask(Q.positive(x), Q.real(x) & ~Q.negative(x) & ~Q.zero(x))
+        True
+        >>> ask(Q.positive(1))
+        True
+        >>> ask(Q.nonpositive(I))
+        False
+        >>> ask(~Q.positive(I))
+        True
+
+        """
+        return Predicate('positive')
+
+    @memoize_property(predicate_storage)
+    def negative(self):
+        r"""
+        Negative number predicate.
+
+        ``Q.negative(x)`` is true iff ``x`` is a real number and :math:`x < 0`, that is,
+        it is in the interval :math:`(-\infty, 0)`.  Note in particular that negative
+        infinity is not negative.
+
+        A few important facts about negative numbers:
+
+        - Note that ``Q.nonnegative`` and ``~Q.negative`` are *not* the same
+          thing. ``~Q.negative(x)`` simply means that ``x`` is not negative,
+          whereas ``Q.nonnegative(x)`` means that ``x`` is real and not
+          negative, i.e., ``Q.nonnegative(x)`` is logically equivalent to
+          ``Q.zero(x) | Q.positive(x)``.  So for example, ``~Q.negative(I)`` is
+          true, whereas ``Q.nonnegative(I)`` is false.
+
+        - See the documentation of ``Q.real`` for more information about
+          related facts.
+
+        Examples
+        ========
+
+        >>> from sympy import Q, ask, symbols, I
+        >>> x = symbols('x')
+        >>> ask(Q.negative(x), Q.real(x) & ~Q.positive(x) & ~Q.zero(x))
+        True
+        >>> ask(Q.negative(-1))
+        True
+        >>> ask(Q.nonnegative(I))
+        False
+        >>> ask(~Q.negative(I))
+        True
+
+        """
+        return Predicate('negative')
+
+    @memoize_property(predicate_storage)
+    def zero(self):
+        """
+        Zero number predicate.
+
+        ``ask(Q.zero(x))`` is true iff the value of ``x`` is zero.
```

Comment:
Since having an assumption for a single value seems odd to most people, it might be worth adding some kind of note here about how this is useful for deductions, for instance, `ask(Q.zero(x*y), Q.zero(x) | Q.zero(y))`. It's also here for the real trichotomy (every real is negative, positive, or zero).